migrations/20191031175708-RenameUserIdColumnToPaymentId.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
This file is mis-named.    should be ToPlanterId

Reply:
Of course. My apologies. Late in the day, I'll review my work better.

Reply:
why did we change it to planter cause technically its still the user of the app (farmer, caretaker, maybe planter)